src/scripts/buttonClickHandlers.js

```diff
@@ -0,0 +1,124 @@
+import { getSavedArray, changeSavedArray } from './utilityFunctions';
+import { renderEditForm } from './renderEditForm';
+
+const getItemIndex = (clickedItem, array) => {
+    return Array.from(array).indexOf(clickedItem);
+};
+
+const addItemObject = value => {
+    return {
+        value: value,
+        progress: 'In Progress'
```

Comment:
zamiast progress może po prostu: status? :)
```suggestion
        status: 'In Progress'
```